osidb/models.py

```diff
@@ -559,6 +559,30 @@ class FlawMajorIncident(models.TextChoices):
 
         INVALID = "INVALID"
 
+    class FlawNistCvssValidation(models.TextChoices):
+        """
+        This flag determines the status of feedback loop between NIST and RH
+        requesting NVD CVSSv3 rescore for the CVE of this flaw.
+        The flag states have the following meaning:
+
+        * ( ) <unset>: No CVSSv3 rescore request was sent to NIST.
+        * (?) "requested": CVSSv3 rescore request was sent to NIST and the
+              feedback loop between NIST and RH is still opened. This status
+              is automatically set by SFM2 after sending request through Email
+              NIST functionality.
+        * (+) "accepted": CVSSv3 rescore request feedback loop between NIST and
+              RH was closed and NIST fully or partially accepted the request
+              lowering NVD score below 7.0.
+        * (-) "rejected": CVSSv3 rescore request feedback loop between NIST and
+              RH was closed and NIST rejected the request or accepted it only
+              partially so the NVD CVSSv3 score is still above or equal 7.0.
+        """
+
+        NOVALUE = ""
+        REQUESTED = "REQUESTED"
+        ACCEPTED = "ACCEPTED"
```

Comment:
Could you please use `APPROVED` instead of `ACCEPTED` to make it similar to major incident state?

Reply:
Changed.